### PR TITLE
[SYCLomatic][CMake] Refine CMake helper function file "dpct.cmake" a part of user migrated code project instead of part of DPCT/SYCLomatic package

### PR DESCRIPTION
**Pull request by tomflinda**
Signed-off-by: chenwei.sun <chenwei.sun@intel.com>

### CODE REVIEW

**Review by zhiweij1 — CHANGES_REQUESTED** [review]
Please make dpct.cmake as a dependency of the execution of processFiles.py

>> chenwei: added.

**Review by zhiweij1 — APPROVED** [review]

File: clang/test/dpct/cmake_migration/case_001/expected.txt [hunk 1]
```diff
@@ -3,10 +3,7 @@ cmake_minimum_required(VERSION 3.24)
 #CMake_minimum_required(VERSION 3.10)
 project(foo-bar LANGUAGES CXX )
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsycl")
-find_program(dpct_bin_path NAMES dpct PATHS)
-get_filename_component(bin_path_of_dpct ${dpct_bin_path} DIRECTORY)
-set(dpct_cmake_file_path "${bin_path_of_dpct}/../cmake/dpct.cmake")
-include(${dpct_cmake_file_path})
+include(./dpct.cmake)
```
Comment (zhimingwang36) [inline clang/test/dpct/cmake_migration/case_001/expected.txt]: let's use a variable name instead of using "./dpct.cmake" directly.

Comment (tomflinda) [reply clang/test/dpct/cmake_migration/case_001/expected.txt]: Okay

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**